sdk/go/common/util/executable/executable.go

```diff
@@ -55,3 +64,15 @@ func FindExecutable(program string) (string, error) {
 
 	return "", errors.Errorf(unableToFindProgramTemplate, program)
 }
+
+func getPotentialPaths(goPath string, os string) []string {
```

Comment:
I think this should be called `splitGoPath` - it would make it more clear to me about what it does